routing/routing_session.cpp

```diff
@@ -589,9 +589,23 @@ void RoutingSession::OnTrafficInfoClear()
 
 void RoutingSession::OnTrafficInfoAdded(TrafficInfo && info)
 {
+  ASSERT_EQUAL(kSpeedGroupThresholdPercentage[static_cast<size_t>(SpeedGroup::G5)], 100, ());
```

Comment:
It's an index (in array) type. The same thing is done in traffic/speed_groups.hpp ```extern uint32_t const kSpeedGroupThresholdPercentage[static_cast<size_t>(SpeedGroup::Count)];```

Do you think int is better here?